src/WebApi/Filters/DefaultExceptionHandlerExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace SSW.CleanArchitecture.WebApi.Filters;
+
+public static class DefaultExceptionHandlerExtensions
+{
+    public static void UseDefaultExceptionHandler(this WebApplication app)
```

Comment:
```suggestion
    // NOTE: Usually app.UseExceptionHandler() is used in conjunction with IExceptionHandler, but this also requires the ProblemDetails service to be registered so that it can be used for unhandled exceptions.  This is not needed as we will handle all exceptions either via this middleware or KnownExceptionsHandler.
    public static void UseDefaultExceptionHandler(this WebApplication app)
```